src/pybind/mgr/dashboard/frontend/package-lock.json

```diff
@@ -4170,12 +4170,14 @@
         "balanced-match": {
           "version": "1.0.0",
           "bundled": true,
-          "dev": true
+          "dev": true,
```

Comment:
Please discard all the changes in this file.
For some reason, sometimes npm adds this changes. This can be removed by running 'npm install' again.